jaxrs/WFLY-13122_EAP7-1494_Add_Json_Merge_Patch_support.adoc

```diff
@@ -0,0 +1,63 @@
+= Add Json Merge Patch support
+:author:            Jim Ma
+:email:             ema@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          json merge patch,resteasy,jaxrs
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+This feature is to support JSON Merge Patch spec as described in: https://tools.ietf.org/html/rfc7386
+With Json merge patch support, the client can send a json http request with Content-Type "application/merge-patch+json",
+and the json merge request will be directly apply to the target resource with the merge function as spec described.
+
+
+== Issue Metadata
+
+=== Issue
+
+* {issue-base-url}/EAP7-1494 [EAP7-1494]
+* {issue-base-url}/WFLY-13412 [WFLY-13412]
+
+=== Related Issues
+
+* {issue-base-url}/RESTEASY-2567 [RESTEASY-2567]
+
+=== Dev Contacts
+
+* mailto:ema@redhat.com[Jim Ma]
+
+=== QE Contacts
+
+* Petr Kremensky
+
+=== Testing By
+
+[ ] Engineering
+
+[x] QE
+
+=== Affected Projects or Components
+
+* RESTEasy, Wildfly
+
+=== Other Interested Projects
+https://tools.ietf.org/html/rfc7386
+
+== Requirements
+
+=== Hard Requirements
+
+* RESTEasy includes this feature support and is installed in WFLY/EAP
```

Comment:
This analysis should include the fact that we need to backport the feature as per https://issues.redhat.com/browse/RESTEASY-2567 since now it reads that no work is needed.

Reply:
This is already included in 3.15.9.Final. There is no work is needed for EAP.

Reply:
@jimma My bad, I see the test now in 3.15 branch https://github.com/resteasy/resteasy/blob/3.15/testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/resource/patch/StudentPatchTest.java So the https://issues.redhat.com/browse/RESTEASY-2567 is outdated?

Reply:
Sorry, my mistake. Checked it again and json merge patch isn't backported in 3.15.x, but this is already included since 4.1.x ,  should be already included in EAP8 and no work is need for EAP8.